language/move-stdlib/src/natives/type_name.rs

```diff
@@ -28,7 +28,7 @@ fn native_get(
     debug_assert!(arguments.is_empty());
 
     let type_tag = context.type_to_type_tag(&ty_args[0])?;
-    let type_name = type_tag.to_canonical_string();
+    let type_name = type_tag.to_canonical_string(false);
```

Comment:
The type_name should be `with_prefix` true?

Reply:
make native type name and debug to_canonical_string with prefix true